docs/packages/Package Development/package development main.rst

```diff
@@ -49,11 +47,14 @@ We recommend `Hitchhiker's Guide to Python <http://docs.python-guide.org/en/late
 
       Don't forgot to follow a good `structure <http://docs.python-guide.org/en/latest/writing/structure/>`_
 
-**7. Add a page to Mesa Packages WIKI page**
+**7. Let people know about your package on the MESA wiki page**
 
-      - `Build your package page <https://github.com/projectmesa/mesa/wiki>`_  
-      - Edit the Mesa Packages wiki page to add your package name
+      - `MESA Wiki Page <https://github.com/projectmesa/mesa/wiki>`_  
    
+You want to do even more. The authoritative guide for python package development is through the `Python Packaging User Guide <https://packaging.python.org/>`_. This will take you through the entire process necessary for getting your package on the Python Package Index.
```

Comment:
This is great! Maybe add a little more about why this is different than just putting on the wiki.